modules/core/shared/src/main/scala/lucuma/core/math/Angle.scala

```diff
@@ -487,65 +485,12 @@ trait AngleOptics extends OpticsHelpers { this: Angle.type =>
  * @see
  *   The helpful [[https://en.wikipedia.org/wiki/Hour_angle Wikipedia]] article.
  */
-final class HourAngle private (µas: Long) extends Angle(µas) {
-
-  // Sanity checks … should be correct via the companion constructor.
-  assert(toMicroarcseconds % 15 === 0, s"Invariant violated. $µas isn't divisible by 15.")
-
-  /**
-   * Flip this HourAngle by 12h. This is logically identical to the superclass implementation and
-   * serves only to refine the return type. Exact, invertible.
-   * @group Transformations
-   */
-  override def flip: HourAngle =
-    this + HourAngle.HourAngle12
-
-  /**
-   * Additive inverse of this HourAngle (by mirroring around the 0-12h axis). This is logically
-   * identical to the superclass implementation and serves only to refine the return type. Exact,
-   * invertible.
-   * @group Transformations
-   */
-  override def unary_- : HourAngle =
-    HourAngle.fromMicroseconds(-toMicroseconds)
-
-  /**
-   * This `HourAngle` in microseconds. Exact.
-   * @group Conversions
-   */
-  def toMicroseconds: Long =
-    toMicroarcseconds / 15
-
-  /**
-   * This `HourAngle` in decimal hours. Approximate.
-   * @group Conversions
-   */
-  def toDoubleHours: Double =
-    toMicroseconds.toDouble / HourAngle.µsPerHour
-
-  /**
-   * Sum of this HourAngle and `ha`. Exact, commutative, invertible.
-   * @group Operations
-   */
-  def +(ha: HourAngle): HourAngle =
-    HourAngle.fromMicroseconds(toMicroseconds + ha.toMicroseconds)
-
-  /**
-   * Difference of this HourAngle and `ha`. Exact, invertible.
-   * @group Operations
-   */
-  def -(ha: HourAngle): HourAngle =
-    HourAngle.fromMicroseconds(toMicroseconds - ha.toMicroseconds)
-
-  /** String representation of this HourAngle, for debugging purposes only. */
-  override def toString: String =
-    HourAngle.fromStringHMS.taggedToString("HourAngle", this)
-
-}
+opaque type HourAngle <: Angle = Long
```

Comment:
In the past `HourAngle` extended `Angle` presumably to reuse some of the methods though HourAngle requires the value to be divisible by 15.
I suppose we may want to pass `HourAngle` to methods taking `Angle?

Reply:
Cool, I didn't know you could do that in an `opaque type` definition.  What happens now  when you try to, say, add an `Angle` to an `HourAngle`?  Is it any different from what happened in the previous version?

Reply:
It should work. I'll check if there is a test and add one if not